From b6eda5abff85cb5764900d1da944096ece4b657c Mon Sep 17 00:00:00 2001
From: Jeremy Soller <jackpot51@gmail.com>
Date: Wed, 8 Jun 2022 07:59:46 -0600
Subject: [PATCH] hwdb: Add HP Dev One

This enables the microphone mute and programmable hotkey for the HP Dev
One.

(cherry picked from commit 295e0291bff52a12d4ce7d7481c4da464002fbb6)

Related: #2087778
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 7204e4ea2a..51c631929e 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -764,6 +764,11 @@ evdev:name:gpio-keys:phys:gpio-keys/input0:ev:23:dmi:*:svnHewlett-Packard:pnHPSt
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP:pnOMEN*:pvr*
  KEYBOARD_KEY_a1=!calc
 
+# HP Dev One
+evdev:atkbd:dmi:*:rvnHP:rn8A78:*
+ KEYBOARD_KEY_81=f20                                    # Fn+F8; Microphone mute button
+ KEYBOARD_KEY_f9=prog1                                  # Fn+F12; Programmable hotkey
+
 ##########################################################
 # Huawei
 ##########################################################
